src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -18,26 +18,25 @@
             + " the contact list.\n"
-            + "search KEYWORD [MORE_KEYWORDS]: Finds all contacts that contain the KEYWORD(s).\n"
-            + "delete INDEX: Deletes the specified person from the address book.\n"
-            + "clear: Clears all entries from the contact list.\n"
-            + "archvie INDEX: Archives the specified employee’s contact detail from tCheck.\n"
-            + "archvie archive ALL: Archives all employees’ contact detail from tCheck.\n"
-            + "archive list: Shows a list of all archived employees’ contact details in tCheck.\n"
-            + "set INGREDIENT_NAME AMOUNT: Sets the level for one type of ingredient individually.\n"
-            + "set all ingredients AMOUNT: Sets all ingredient levels to a standard default amount.\n"
-            + "set ingredients m/MILK b/BLACK_TEA s/SUGAR o/BUBBLE: Enters all current ingredient levels.\n "
-            + "ingredient single INGREDIENT_NAME: Prints a particular type of ingredient's ingredient level. \n"
-            + "ingredient resetAll: Sets all ingredient levels to 0 by updating the database when the command is "
+            + "c-search KEYWORD [MORE_KEYWORDS]: Finds all contacts that contain the KEYWORD(s).\n"
+            + "c-delete INDEX: Deletes the specified person from the address book.\n"
+            + "c-clear: Clears all entries from the contact list.\n"
+            + "c-archvie INDEX: Archives the specified employee’s contact detail from tCheck.\n"
+            + "c-archive-all: Archives all employees’ contact detail from tCheck.\n"
+            + "c-archive-list: Shows a list of all archived employees’ contact details in tCheck.\n"
+            + "c-unarchive INDEX: Unarchives the specified employees’ contact detail.\n"
+            + "i-set i/INGREDIENT_NAME m/AMOUNT: Sets the level for one type of ingredient individually.\n"
+            + "i-set-default: Sets all ingredient levels to a standard default amount.\n"
+            + "i-view-single i/INGREDIENT_NAME: Prints a particular type of ingredient's ingredient level. \n"
+            + "i-reset-all: Sets all ingredient levels to 0 by updating the database when the command is "
             + "entered.\n"
-            + "ingredient list: Prints the ingredient levels for all ingredient types retrieved from the database.\n"
-            + "sales list.: Shows a list of all types of drinks sold for the current day.\n"
-            + "sales YYYY-MM-DD: Shows a sales record for each type of mile tea on a specific date.\n"
-            + "sales A/NUM B/NUM C/NUM ...: Asks the user to enter the number of each type of drink sold for the "
+            + "i-list: Prints the ingredient levels for all ingredient types retrieved from the database.\n"
+            + "s-list.: Shows a list of all types of drinks sold for the current day.\n"
+            + "s-set: A/NUM B/NUM C/NUM ...: Asks the user to enter the number of each type of drink sold for the "
```

Comment:
The command is `s-update`. Additional there should not be a colon after the command word.

Reply:
Noted with thanks